### PR TITLE
Changes in casing of the properties in Auth and Pre Auth flow.

### PR DESCRIPTION
**Pull request by nithyaganeshng**
Properties were changed to camel casing for Auth and Pre Auth flows.

**Validation:**
Please include here how it was verified that the PR works as intended.
Tested via demo app.

**Type of change:**
- [x] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [x] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/NetworkErrorParser.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ internal object NetworkErrorParser {
                 code = (error?.get("code") as? JsonPrimitive)?.content
             }
         } catch (ex: Exception) {
-            SdkLog.d("Parsing error response canceled. Json: $errorBody", ex)
+            SdkLog.i("Parsing error response canceled. Json: $errorBody", ex)
```
Comment (logangirvin) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/NetworkErrorParser.kt]: This _may_ include PII.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/NetworkErrorParser.kt [hunk 2]
```diff
@@ -59,7 +59,7 @@ internal object NetworkErrorParser {
                 msg = (error?.get("message") as? JsonPrimitive)?.content
             }
         } catch (ex: Exception) {
-            SdkLog.d("Parsing error response canceled. Json: $errorBody", ex)
+            SdkLog.i("Parsing error response canceled. Json: $errorBody", ex)
```
Comment (logangirvin) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/NetworkErrorParser.kt]: Likewise may include PII.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt [hunk 3]
```diff
@@ -39,7 +41,11 @@ internal class OpenIdURLRequestResolver(val libraryConfiguration: LibraryConfigu
         if (verifiedIdRequestInput !is VerifiedIdRequestURL) throw UnSupportedVerifiedIdRequestInputException(
             "Provided VerifiedIdRequestInput is not supported."
         )
-        return resolveOpenId4VCIRequest(verifiedIdRequestInput)
+        if (libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_ACCESS_TOKEN)
+            || libraryConfiguration.isPreviewFeatureEnabled(PreviewFeatureFlags.FEATURE_FLAG_OPENID4VCI_PRE_AUTH)
+        )
+            return resolveOpenId4VCIRequest(verifiedIdRequestInput)
+        return OpenIdResolver.getRequest(verifiedIdRequestInput.url.toString(), preferHeader)
```
Comment (logangirvin) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt]: Comment left on the other PR but I had to change this in oct24rc for extensions to parse requests.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/NetworkErrorParser.kt (hunk 2 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/util/NetworkErrorParser.kt]: Fixed it.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt (hunk 3 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/resolvers/OpenIdURLRequestResolver.kt]: We need this check to determine what protocol to use while fetching the request. But moved it to a different location (right before fetching the request).

**Review by logangirvin — APPROVED** [review]

**Pull request merged**